src/main/java/net/atos/client/zgw/shared/ZGWApiService.java

```diff
@@ -319,6 +323,28 @@ private Resultaattype filterResultaattype(List<Resultaattype> resultaattypes, fi
     }
 
     private void berekenArchiveringsparameters(final UUID zaakUUID) {
-        // ToDo:
+        final Zaak patch = new Zaak();
+        final Zaak zaak = zrcClientService.readZaak(zaakUUID); // refetch to get the einddatum
+        final Resultaattype resultaattype = ztcClientService.readResultaattype(zrcClientService.readResultaat(zaak.getResultaat()).getResultaattype());
+        patch.setArchiefnominatie(resultaattype.getArchiefnominatie());
```

Comment:
archiefnominatie wort al gezet door OperZaak bij het afsluiten van de zaak. Kijk maar eens naar de laatste methode in Zaak.java. We hebben al eerder geconcudeerd dat het al dan niet gevuld zijn van archiefnominatie een goed test was om te kijken of de zaak gesloten is of niet. Je hoeft archiefnominatie hier dus geen waarde te geven.